protoactor/protos_pb2.py

```diff
@@ -0,0 +1,267 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
This auto generated file will be included in the code. Travis CI will not modify the PR and not include this file into commit. When someone fetchs project will need to generate it self.